chart/values.yaml

```diff
@@ -592,6 +592,28 @@ workers:
     # This configuration will be ignored if PGBouncer is not enabled
     usePgbouncer: true
 
+  # Allow HPA (KEDA must be disabled).
+  hpa:
+    enabled: false
+
+    # Minimum number of workers created by HPA
+    minReplicaCount: 0
+
+    # Maximum number of workers created by HPA
+    maxReplicaCount: 10
+
+    # Specifications for which to use to calculate the desired replica count
+    metrics: |-
```

Comment:
IMHO this should be an array of metrics instead of string.

Reply:
Hello @hussein-awala, thanks for reviewing. I added metrics as string to support `tpl` blocks, just in case someone wants to use parameters from `values.yaml`, for example:
```yaml
metrics: |-
  - type: Pods
    pods:
      metric:
        name: executor_running_tasks
      target:
        type: AverageValue
        averageValue: {{ .Values.config.celery.worker_concurrency }}
```
Do you see it feasible?

Reply:
Metrics block set as array.